src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -15,14 +16,15 @@ public class AuthenticationController {
     private final ResponseDtoMapper<UserResponseDto, User> userDtoResponseMapper;
 
     public AuthenticationController(AuthenticationService authService,
-            ResponseDtoMapper<UserResponseDto, User> userDtoResponseMapper) {
+                                    ResponseDtoMapper<UserResponseDto,
+                                            User> userDtoResponseMapper) {
         this.authService = authService;
         this.userDtoResponseMapper = userDtoResponseMapper;
     }
 
     @PostMapping("/register")
-    public UserResponseDto register(@RequestBody UserRequestDto requestDto) {
-        User user = authService.register(requestDto.getEmail(), requestDto.getPassword());
+    public UserResponseDto register(@RequestBody @Valid UserRegistrationDto registrationDto) {
+        User user = authService.register(registrationDto.getEmail(), registrationDto.getPassword());
```

Comment:
I'm not sure we are supposed to create new Dto, because task was: `In the UserRequestDto add a repeatPassword field.`
But maybe it's ok.